lib/global_id/locator.rb

```diff
@@ -106,7 +106,9 @@ def use(app, locator = nil, &locator_block)
 
       private
         def locator_for(gid)
-          @locators.fetch(normalize_app(gid.app)) { default_locator }
+          @locators.fetch(normalize_app(gid.app)) do
+            gid.model_class.respond_to?(:unscoped) ? ACTIVE_RECORD_LOCATOR : DEFAULT_LOCATOR
```

Comment:
Actually we shouldn't do class detection but feature detection here. For example mongoid has the same default_scope / unscoped pattern (see [here](http://mongoid.org/en/mongoid/docs/querying.html#scoping)).